chroma-manager.conf.template

```diff
@@ -78,7 +86,7 @@ server {
     }
 
     location ~ ^/$ {
-        return 301 https://$http_host/ui;
+        return 301 https://$http_host/ui2;
```

Comment:
Should this be 302 until it can be changed back to `/ui`?

Reply:
Maybe 307?

Reply:
Updated to 307